Workspace/src/rover_autonomy/src/ball_tracking.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
```

Comment:
Please don't merge this in yet, I don't think it works well right? Also, we're re-doing it